app/views/layouts/groups.html.erb

```diff
@@ -31,12 +31,16 @@
               selected: @current_page == t(:"groups.sidebar.history").downcase
             ) %>
           <% end %>
-          <% if allowed_to?(:update?, @group) %>
-            <%= render section.with_item(
-              url: edit_group_path(@group),
-              icon: "cog_6_tooth",
-              label: t(:"groups.sidebar.settings"),
-              selected: @current_page == t(:"groups.sidebar.settings").downcase
+          <%= render section.with_multi_level_menu(title: t(:"groups.sidebar.settings"), icon: "cog_6_tooth") do |multi_level_menu| %>
```

Comment:
**nitpick (blocking):** I think this whole section should be surrounded with `allowed_to?(:update?, @group)` as non `Maintainer` or `Owner' should not see this section.